src/app/core/components/top-right-menu/top-right-menu.component.ts

```diff
@@ -10,13 +13,22 @@ export class TopRightMenuComponent {
 
   readonly menuItems = [
     { label: 'Profile', icon: 'pi pi-user', routerLink: [ 'groups', 'me' ] },
+    ...(!appConfig.production
+      ? [{ label: 'Invalidate token', icon: 'pi pi-refresh', command: (): void => this.invalidateToken() }]
+      : []
+    ),
     { label: 'Log out', icon: 'pi pi-power-off', command: ():void => this.sessionService.logout() },
   ]
 
-  sessions$ = this.sessionService.session$;
+  userLogin$ = this.sessionService.session$.pipe(map(session => session?.user.login), distinctUntilChanged())
```

Comment:
so I assume it can follow `userProfile$` instead.

Reply:
I think we can't because the user profile filters out the empty sessions. If the token is invalidated and no session is started for any reason (bad connection, etc.), the previous login would be displayed. Now I'm not sure whether this specific situation might happen, so I assumed it was a possibility.

Reply:
It might happen but the UI will lock with the error, so that's not very problematic. Your point is correct as well.